lib/shared/models/flash-state.js

```diff
@@ -123,9 +123,7 @@ exports.setProgressState = (state) => {
 
       speed: _.attempt(() => {
         if (_.isNumber(state.speed) && !_.isNaN(state.speed)) {
-          // Preserve only two decimal places
-          const PRECISION = 2
-          return _.round(units.bytesToMegabytes(state.speed), PRECISION)
+          return units.bytesToClosestUnit(state.speed) + '/s'
```

Comment:
No need to `_.attempt()` here anymore, since it's being checked whether it's a number. Also might want to use `.isFinite()` here instead of `!_.isNaN()`

Reply:
Good call